site/source/tutorials/007_theming/index.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: tutorial
+title: Theming
+overview: In this tutorial, you will learn how to use the Dojo 2 theming and styling system to add a theme to the sample application.
+---
+
+{% section 'first' %}
+
+# Theming
+
+## Overview
+This tutorial will extend on previous tutorials where we created the basic biz-e-corp application. In this tutorial, we will adapt the app to allow it to be themed, write a theme and apply it to our application. This will be done using the powerful theming system that is baked into Dojo 2.
+
+## Prerequisites
+You can [download](../assets/007_themeing-initial.zip) the demo project and run `npm install` to get started.
+
+The `@dojo/cli` command line tool should be installed globally. Refer to the [Dojo 2 local installation](../000_local_installation/) article for more information.
+
+You also need to be familiar with TypeScript as Dojo 2 uses it extensively. For more information, refer to the [TypeScript and Dojo 2](../comingsoon.html) article.
+
+{% section %}
+
+## Making our widgets themeable
+
+{% task 'Make our widgets themeable' %}
+
+In order to theme our widgets, we must ensure that they each extent `ThemeableMixin` and should change their top level class name to `root`. `ThemeableMixin` provides us with a `this.classes` function that can lookup class names in the provided `theme` file and applies classnames appropaitely. We change the top level classnames to `root` in order to provice a predictable way to target the outernode of a iwgdte.
+
+{% instruction 'Add the following to `Banner.ts`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/Banner.ts' lines:3-13 %}
+
+{% aside 'Reminder' %}
+If you cannot see the application, remember to run `dojo build -w` to build the application and start the development server.
+{% endaside %}
+
+This `Banner` widget will now have access to the class files in `banner.m.css` and can receive a `theme`. We use the `root` class here as detailed above to ensure that the theme we create can easily target the correct node.
+
+{% instruction 'Create a new css file for `Banner` named `banner.m.css`.' %}
+
+We will create an empty `root` class for now as our base theme does not require any styles to be added to the `banner`.
+
+{% include_codefile 'demo/finished/biz-e-corp/src/styles/banner.m.css' lang:css %}
+
+{% instruction 'Now, lets look at changing `WorkerForm`' %}
+
+`WorkerForm` already uses the `ThemeableMixin` and has a `workerForm` class on it's root node. Lets go ahead and change that to a `root` class, but while we're there, we will create a `rootFixed` class too and apply that as a `fixed` class to the root node. This passes a classname to the theming system that cannot be changed or overridden via a theme and ensures that structured or nested styles are not lost when a theme is used.
```

Comment:
does this introduce fixed classes before explaining what they are? should we introduce the types of class that can be applied before?

Reply:
Added an aside.